bescort.lic

```diff
@@ -390,7 +390,13 @@ class Bescort
       path.each do |movement|
         move(movement)
         waitrt
-        fput('collect rock')
+        case bput('collect rock','You manage to collect a pile', 'The room is too cluttered')
```

Comment:
What would you think about moving the collect logic into `common.lic`, similar to how `forage?` is done? I see some overlap here with the collect logic in `crossing-training`.